src/ils.cpp

```diff
@@ -101,6 +101,66 @@ usage() {
+        "working directory.",
+        " ",
+        "The long format (given by -l) will display some more information:",
+        "    $ ils -l",
+        "    /tempZone/public:",
+        "      rods              0 demoResc     41943040 2021-04-29.17:57 & big",
+        "      rods              2 comp;arch    40236581 2021-04-19.03:00 X big",
+        "      alice             2 repl1;child1          283 2021-04-29.17:54 ? foo",
+        "      alice             3 repl1;child2          283 2021-04-29.17:56 ? foo",
+        "      alice             4 repl1;child3          283 2021-04-29.17:57 ? foo",
+        "This displays replica information and system metadata for the replicas of",
+        "the data objects in the current working directory. This information includes",
+        "the owner of the data object and the replica's number, size, last modified",
+        "time, and status.",
+        " ",
+        "The very long format (given by -L) will display even more information:",
+        " ",
+        "    $ ils -L",
+        "    /tempZone/public:",
+        "      rods              0 demoResc     41943040 2021-04-29.17:57 & big",
+        "        sha2:gKNyEYjkAhiwiyZ3a8U72ugeR4T/9x1xRQoZcxnLoRM=    generic    /var/lib/irods/Vault/home/rods/big",
+        "      rods              2 comp;arch    40236581 2021-04-19.03:00 X big",
+        "        sha2:KNyEYjkhiwyZ3a8U72ugeR4T/9x1xRQoZcxnLoRMigg=    generic    /var/lib/irods/Vault/home/rods/big",
+        "      alice             2 repl1;child1          283 2021-04-29.17:54 ? foo",
+        "            generic    /var/lib/irods/child1vault/home/rods/foo",
+        "      alice             3 repl1;child2          283 2021-04-29.17:56 ? foo",
+        "            generic    /var/lib/irods/child2vault/home/rods/foo",
+        "      alice             4 repl1;child3          283 2021-04-29.17:57 ? foo",
+        "            generic    /var/lib/irods/child2vault/home/rods/foo",
+        "This displays even more replica information and system metadata such as each ",
+        "replica's checksum, data type, and physical location in the storage resource.",
+        " ",
+        "The different replica statuses are represented by the following characters ",
+        "which can be seen next to the data object name on the right of an ils -l/-L:",
+        "   0. X: stale        - The replica is no longer known to be good. Usually",
+        "                        indicates that a more recently written-to replica exists.",
+        "                        This does not necessarily mean that the data is incorrect",
+        "                        but merely that it may not reflect the \"truth\" of this",
+        "                        data object.",
+        "   1. &: good         - The replica is good. Usually the latest replica to be",
+        "                        written. When a replica is good, all bytes are understood",
```

Comment:
and system metadata (size and checksum, if present)

Reply:
Done